__init__.py

```diff
@@ -542,9 +550,21 @@ def get_dashboard_cards(self, message=None):
         cards = self.dashboard_handler.get_collection()
         collection = {"collection": cards}
         return collection
-    
-
 
+    ######################################################################
+    # Handle Screenshot
+
+    @intent_file_handler("take.screenshot.intent")
+    def take_screenshot(self, message):
+        folder_path = self.screenshot_folder
+        if not folder_path:
+            folder_path = os.path.expanduser('~') + "/Pictures"
+        self.bus.emit(Message("ovos.display.screenshot.get", {"folderpath": folder_path}))
```

Comment:
Screenshot reports success but fails if the directory doesn't exist, so there should be a check here

```python
os.makedirs(folder_path, exist_ok=True)
```

Reply:
A check has been added to check if folder exist, if it does not then it will write it to /tmp